Makefile

```diff
@@ -23,11 +26,14 @@ install-tools:
 	hack/install-kubebuilder-tools.sh
 
 test: build install-tools
-	hack/test-go.sh
+	hack/test-go.sh -u $(USE_STATIC_CHECK)
```

Comment:
this could just be static, like:

```
hack/test-go.sh --use-static-check
```

(or if you flip the logic, then use `--skip-static-check` in the `upstream-test` make target.